src/Sylius/Bundle/ResourceBundle/Controller/ResourceController.php

```diff
@@ -461,4 +463,24 @@ protected function isGrantedOr403($permission)
             throw new AccessDeniedHttpException();
         }
     }
+
+    protected function isApiRequest()
+    {
+        if (!$this->config->isApiRequest()) {
+            return false;
+        }
+
+        $bundles = array(
+            'JMSSerializerBundle',
+            'BazingaHateoasBundle'
```

Comment:
You are not obliged to use `BazingaHateoasBundle` when you build API it is just better!